### ISSUE TITLE
Paginate variable not available for search results

### ISSUE THREAD

**Issue by rustypaul** [statamic/cms#3682]
The paginate variable isn't available on search results. When you enable pagination it correctly limits the number of results returned but doesn't give you the paginate variable to build pagination with. For example, the code below will limit the results to 12, but never shows the paginate partial.

```
{{ search:results paginate="12" }}

   {{ if no_results }}
      <h2>No results.</h2>
   {{ else }}

      <a href="{{ url }}" class="result">
         <h2>{{ title }}</h2>
         <p>{{ content | truncate:240 }}</p>
      </a>

      {{ paginate }}
         {{ partial src="partials.paginate" }}
      {{ /paginate }}

   {{ /if }}

{{ /search:results }}
```

Statamic 3.1.12 Pro
Laravel 6.20.26
PHP 7.4.4
spatie/statamic-responsive-images 2.5.4
statamic/ssg 0.6.0

**Install method (choose one):**
- Starter kit (cool writings)

**Comment by kesse** [comment]
The support for search pagination was mentioned in the change log for 3.0.30 
https://github.com/statamic/cms/blob/v3.0.30/CHANGELOG.md

Would be greate if it could be fixed since I'm using it on my site.

**Comment by beck24** [comment]
Just found this as well, this was working (though poorly documented) in Statamic 2, currently upgrading to Statamic 3 and we now have no pagination for searching.

**Comment by jasonvarga** [comment]
This is fixed for 3.1.26. You'll need to add `as="results"` to your tag.
